src/bin/coldsnap/main.rs

```diff
@@ -35,9 +32,59 @@ async fn main() {
 
 async fn run() -> Result<()> {
     let args: Args = argh::from_env();
+    let config: aws_config::ConfigLoader = match (args.region, &args.profile) {
+        (Some(region), Some(_profile)) => {
+            //region option passed in
+            aws_config::from_env().region(Region::new(region))
+        }
+        (Some(region), None) => {
+            //region option passed in
+            aws_config::from_env().region(Region::new(region))
+        }
```

Comment:
I think this works (Also nit, strive for comment format consistency: a space after the `//` and it looks like other comments in the lib start with upper case):

```suggestion
        (Some(region), _) => {
            // Region option passed in
            aws_config::from_env().region(Region::new(region))
        }
```

Reply:
Thanks!